mkosi.extra/root/.gdbinit

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+set debuginfod enabled off
```

Comment:
if you are adding this file, how about setting the source code dir as well?

Reply:
Not required it's baked into the debuginfo